README.md

```diff
@@ -30,9 +31,17 @@ possible to avoid Docker altogether and run everything directly on the host mach
 
 Before starting the installation process, please have a look at the system requirements and the note on security.
 
+### Networks
+
+There are two blockchain networks related to this project. The test network is
+called Laika. The productive network is called the Trustlines Blockchain. Often
+this gets abbreviated to `tlbc`, especially for technical components. The
+instructions within this documentation primarily focus on the Trustlines
+Blockchain.
```

Comment:
I would rename the section to "TLBC and Laika" or "The Trustlines Blockchain and Laika" or something like that and not talk about networks.

Reply:
:heavy_check_mark: